content/en/docs/setup/additional-setup/pod-security-admission/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Install Istio with Pod Security Admission
+description: Install and use Istio with the Pod Security admission controller..
+weight: 70
+aliases:
+    - /docs/setup/kubernetes/install/pod-security-admission
+    - /docs/setup/kubernetes/additional-setup/pod-security-admission
+keywords: [psa]
+owner: istio/wg-networking-maintainers
+test: yes
+---
+
+Follow this guide to install, configure, and use an Istio mesh with the Pod Security admission controller
+([PSA](https://kubernetes.io/docs/concepts/security/pod-security-admission/)) enforcing the `baseline` [policy](https://kubernetes.io/docs/concepts/security/pod-security-standards/) on namespaces in the mesh.
+
+By default Istio injects an init container, `istio-init`, in pods deployed in
+the mesh. The `istio-init` requires the user or
+service-account deploying pods to the mesh to have sufficient Kubernetes RBAC
+permissions to deploy [containers with the `NET_ADMIN` and `NET_RAW` capabilities](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-capabilities-for-a-container).
+
+However, the `baseline` policy does not include `NET_ADMIN` or `NET_RAW` in its [allowed capabilities](https://kubernetes.io/docs/concepts/security/pod-security-standards/#baseline). Therefore, it is necessary to use Istio mesh with the [Istio Container Network Interface plugin](/docs/setup/additional-setup/cni/). The `istio-cni-node` DaemonSet in the `istio-system` namespace requires `hostPath` volumes to access local CNI directories. Since this is not allowed in the `baseline` policy, the `istio-system` namespace needs to enforce the `privileged` [policy](https://kubernetes.io/docs/concepts/security/pod-security-standards/#privileged).
```

Comment:
```suggestion
However, the `baseline` policy does not include `NET_ADMIN` or `NET_RAW` in its [allowed capabilities](https://kubernetes.io/docs/concepts/security/pod-security-standards/#baseline). To utilize these capabilities without enforcing the `privileged` policy in all meshed namespaces, it is necessary to use Istio mesh with the [Istio Container Network Interface plugin](/docs/setup/additional-setup/cni/). The `istio-cni-node` DaemonSet in the `istio-system` namespace requires `hostPath` volumes to access local CNI directories. Since this is not allowed in the `baseline` policy, the `istio-system` namespace needs to enforce the `privileged` [policy](https://kubernetes.io/docs/concepts/security/pod-security-standards/#privileged).
```

Reply:
That works for me!